template/checkNothing/checkNothing.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package checkNothing;
+
+import "pkg/adapter/template/TemplateExtensions.proto";
+
+option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_CHECK;
```

Comment:
I don't remember being asked to do this. But it would probably make sense. If we do this, could we also call the message by the desired name instead of calling 'em all Template? So the message would be called ListEntry.

@guptasu Sunny?